### PR TITLE
Omit the explicit import of dftbp_assert

### PR DESCRIPTION
**Pull request by aradi**

### CODE REVIEW

File: prog/dftb+/include/common.fypp [hunk 1]
```diff
@@ -40,9 +40,12 @@
 #! Check a condition if WITH_ASSERT is True and call assertError if condition is False.
 #:def ASSERT(cond)
   #:if WITH_ASSERT
-  if (.not. (${cond}$)) then
-    call assertError("${_FILE_}$", ${_LINE_}$)
-  end if
+  block
+    use dftbp_assert, only : assertError
+    if (.not. (${cond}$)) then
+      call assertError("${_FILE_}$", ${_LINE_}$)
```
Comment (awvwgk) [inline prog/dftb+/include/common.fypp:46]: While are on it, can we pass the actual condition tested as string to the assertion error as well?

**Review by awvwgk — APPROVED** [review]
Looks good to me. Only one completely unrelated suggestion to the `assertError` routine.

Merging this one will probably require rebasing for most feature branches.

File: prog/dftb+/include/common.fypp (hunk 1 above)
Comment (aradi) [reply prog/dftb+/include/common.fypp:46]: Yes, that would make sense, indeed. I'll add it in a separate PR.

**Comment by aradi** [comment]
Solved in #784 .

**Pull request closed without merging**